gamemode/sh_disguise.lua

```diff
@@ -123,6 +123,8 @@ function GM:PlayerCanDisguiseCurrentTarget(ply)
 	if ply:Team() == 3 then
 		local tr = ply:GetPropEyeTrace()
 		if IsValid(tr.Entity) then
+			if(self:IsModelBanned(tr.Entity:GetModel())) then return false, nil end
```

Comment:
Weird style for this `if`.

Reply:
I've fixed this. It was one of the earliest additions before I figured out the style. Just forgot to adjust this one.

Reply:
If you fixed this, it isn't pushed yet.